codalab/rest/bundles.py

```diff
@@ -120,6 +120,7 @@ def build_bundles_document(bundle_uuids):
     # Include permissions
     for bundle in bundles:
         json_api_include(document, BundlePermissionSchema(), bundle['group_permissions'])
+        bundle['group_permissions'] = sorted(bundle['group_permissions'], key=lambda x: x['group_name'])
```

Comment:
Ah, good to know. If I make the change to the BundleModel query though, I won't need this line anymore, right?